File: app/src/main/java/org/dhis2/utils/jira/JiraIssueField.java
package org.dhis2.utils.jira;

/**
 * QUADRAM. Created by ppajuelo on 11/04/2019.
 */
public class JiraIssueField {

    private JiraField issuetype;
    String summary;
    private JiraField status;

    public JiraField getIssuetype() {
        return issuetype;
    }

    public JiraField getStatus() {
        return status;
    }

    public String getSummary() {
        return summary;
    }

    public class JiraField {
        private String name;
        private int id;

        public int getId() {
            return id;
        }

        public String getName() {
            return name;
        }
    }
}

File: app/src/main/java/org/dhis2/utils/jira/JiraIssueField.kt
package org.dhis2.utils.jira

/**
 * QUADRAM. Created by ppajuelo on 11/04/2019.
 */
class JiraIssueField {

    val issuetype: JiraField? = null
    var summary: String? = null
        internal set
    val status: JiraField? = null

    inner class JiraField {
        val name: String? = null
        val id: Int = 0
    }
}
